code/Model/VirtualPage.php

```diff
@@ -77,8 +79,23 @@ class VirtualPage extends Page
 
     private static $db = [
         "VersionID" => "Int",
+        'CustomMetaDescription' => 'Text',
+        'CustomExtraMeta' => 'HTMLText'
```

Comment:
These come from `SubsitesVirtualPage` (see https://github.com/silverstripe/silverstripe-subsites/pull/589). They make a lot of sense to add here, and not much sense to have explicitly only for the subsites version of this page.